app/swagger/schemas/async_transaction/vet360.rb

```diff
@@ -65,8 +66,9 @@ class Vet360
                     AsyncTransaction::Vet360::AddressTransaction
                     AsyncTransaction::Vet360::EmailTransaction
                     AsyncTransaction::Vet360::InitializePersonTransaction
+                    AsyncTransaction::Vet360::PermissionTransaction
                     AsyncTransaction::Vet360::TelephoneTransaction
-                  ], example: 'AsyncTransaction::Vet360::EmailTransaction'
```

Comment:
is this a bad merge or an intentional change?

Reply:
That was just a change trying to get the code climate to not see this as a duplicate entry - but John approved this - I didn't think the example value mattered - it is just an example isn't it?